cuppa-surefire/src/main/java/org/forgerock/cuppa/maven/surefire/Test.java

```diff
@@ -0,0 +1,16 @@
+package org.forgerock.cuppa.maven.surefire;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Identifies a class as containing Cuppa tests.
+ *
+ * <p>Cuppa tests contained with the annotated class will be added to the test run.</p>
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.TYPE, ElementType.CONSTRUCTOR})
+public @interface Test {
```

Comment:
I think this needs to be moved to `cuppa-core` so that the user does not need to a test-compile-time dependency on `cuppa-surefire`.